src/libPMacc/include/particles/operations/CountParticles.hpp

```diff
@@ -113,23 +203,33 @@ struct CountParticles
      * @return number of particles in defined area
      */
     template<uint32_t AREA, class PBuffer, class Filter, class CellDesc>
-    static uint64_cu countOnDevice(PBuffer& buffer, CellDesc cellDescription, Filter filter)
+    static uint64_cu countOnDevice( PBuffer& buffer, CellDesc cellDescription, Filter filter )
```

Comment:
signature: yes we can (and should; offline result) change this via a static cast before of the var we return.

note: please not in this PR, this would change to much downstream code, refactor in a follow-up